zeno/backend.py

```diff
@@ -506,3 +506,6 @@ def single_inference(self, *args):
         if type(out) == tuple and len(out) == 2:
             return out[0][0]
         return out[0]
+
+    def slice_finder(self, req):
```

Comment:
can you put this in a new file in the folder `processing/` like `processing/slice_finder.py`?